instances/species/ovisAries.jsonld

```diff
@@ -0,0 +1,17 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/species/ovisAries",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Species",
+  "definition": "The species 'Ovis aries' (domestic sheep) is a mammal of the artiodactyla (even-toed ungulates) order and genus ovis (wild and domestic sheep).",
```

Comment:
for consistency with the other species this should only resolve to the family.
"The species *Ovis aries* (domestic sheep) belongs to the family of *bovidae* (bovids)."

Reply:
fixed